esmvaltool/utils/batch-jobs/job_DKRZ-MISTRAL.sh

```diff
@@ -1,29 +1,31 @@
-
 #!/bin/bash -e
 ###############################################################################
 ### BATCH SCRIPT TO RUN THE ESMVALTOOL AT DKRZ MISTRAL
 ### Author: Mattia Righi (DLR)
 ###############################################################################
-#SBATCH --partition=compute
-#SBATCH --time=08:00:00
+#SBATCH --partition=prepost
+#SBATCH --exclusive
+#SBATCH --nodes=1
+#SBATCH --mem=0
+#SBATCH --time=12:00:00
 #SBATCH --mail-type=FAIL,END
-#SBATCH --account=bd0854
+#SBATCH --account=XXXXXX
 #SBATCH --output=job_%j.out.log
 #SBATCH --error=job_%j.err.log
 ###############################################################################
 
 # Submit job with: sbatch job_DKRZ-MISTRAL.sh
 
 # Input arguments
-RECIPE=recipe_perfmetrics_CMIP5.yml
-CONFIG=config-user.yml
+RECIPE= # e.g. recipe_perfmetrics_CMIP5.yml
+CONFIG= # e.g. config-user.yml
 
 # Set environment
 CONDAPATH=  # e.g. /home/soft/miniconda3/
-CONDAENV=   # e.g. $CONDAPATH/envs/esmvaltool/bin
 ESMVALPATH= # e.g. /home/ESMValTool/esmvaltool
 
 # Changes below this line should not be required
 export PATH=$PATH:$CONDAPATH/bin/
 conda info --envs
-$CONDAENV/esmvaltool $ESMVALPATH/recipes/$RECIPE -c $ESMVALPATH/$CONFIG
+
+esmvaltool run --config-file $CONFIG $ESMVALPATH/recipes/$RECIPE
```

Comment:
In principle you could also just use
```
esmvaltool run $RECIPE
```
because esmvaltool will automatically look for installed recipes and the config file in the default location (~/.esmvaltool/config-user.yml).

Reply:
Still, I would like to keep the option of specifying a config file different from the default.